etc/config-template.toml

```diff
@@ -665,7 +665,7 @@
 # write-buffer-size = "128MB"
```

Comment:
I think you should change write-buffer-size, not max-bytes-for-level-base.

Reply:
Sorry, I made a mistake.